src/decays/specializations/H/decay_H_to_AZ.inc

```diff
@@ -19,15 +19,27 @@ double CLASSNAME::get_partial_width<Higgs, Photon, ZBoson>(
       res *= alpha_em_0/alpha_em;
    }
 
+   std::string tag = field_as_string<Higgs>(in_idx) + "-" + field_as_string<Photon>(out1_idx) + "-" + field_as_string<ZBoson>(out2_idx);
    if (flexibledecay_settings.get(FlexibleDecay_settings::print_effc_block)) {
       effhiggscouplings_block_input.push_back(
          {
             25 + 10*(in_idx.size()>0 ? in_idx.at(0) : 0), 22, 23,
             std::sqrt(res/(flux * ps)/(0.5*Sqr(Sqr(mH)-Sqr(mZ)))),
-            field_as_string<Higgs>(in_idx) + "-" + field_as_string<Photon>(out1_idx) + "-" + field_as_string<ZBoson>(out2_idx)
+            tag
          }
       );
    }
 
+   if (static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::call_higgstools)) ||
```

Comment:
See my comment on `static_cast<bool>`

Reply:
Done in c587517522b7b65208aa28eb3ac3b70a03a3ce92.